test/DynamoCoreWpfTests/GuidedTourTests.cs

```diff
@@ -62,7 +62,7 @@ public void GetStartedGuide_ValidateControlNames()
 
             //Find the Get Started guide in the list
             var getStartedGuide = (from guide in testGuide.Guides 
-                                  where guide.Name.ToUpper().Equals("GET STARTED")
+                                  where guide.Name.Equals("User Interface Tour")
```

Comment:
You can use the variable GetStartedGuideName here

Reply:
Awesome suggestion, updated